makefile

```diff
@@ -378,6 +378,18 @@ phony += clean
 gqlgen:
 	go run github.com/99designs/gqlgen generate
 
+swaggo-install:
```

Comment:
Since this is a substantially different thing from just "look at the Markdown in your viewer of choice" with the rest of Pine, can you add more specific instructions in the new hire/dev docs + give a demo to give context for devs to learn more?